ocaml/xapi-types/jbuild

```diff
@@ -37,6 +37,7 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
    uuidm
    xapi-consts
    xapi-stdext-date
+   xcp
```

Comment:
You depend on `xcp` here, and on `xapi-idl` below. Which one is the preferred name?

Reply:
I believe `xcp` is the current library name, `xapi-idl` the opam package. We will need the giant debt PR to fix all names after the jbuilder port is complete